code/src/pages/cart/components/cart-item/CartItem.scss

```diff
@@ -52,10 +52,16 @@
   }
 
   &__price {
-    font-weight: var(--spa-cart-price-font-weight, $fw-extra-bold);
-    font-size: $fs-lg;
     margin-left: spacing(5);
-    min-width: 80px;
+    min-width: 101px;
```

Comment:
Why `101px` and not `100px`?

Reply:
> Why `101px` and not `100px`?

for `1px` it's crushed like space to another row

Reply:
I suggest adding comment for this line, because it arises questions and is striking to the eye